pkg/lib/config/identity.go

```diff
@@ -281,7 +281,8 @@ var _ = Schema.Add("OAuthSSOProviderType", `
 		"azureadb2c",
 		"adfs",
 		"apple",
-		"wechat"
+		"wechat",
+		"mock"
```

Comment:
Do not add `mock` here. The baseline is do not modify existing code to facilitate the e2e test.